docs/2_4_common_schema.adoc

```diff
@@ -449,17 +449,17 @@ _Note that `Hz` is typically used as `Unit.name` for a frequency quantity, but i
 
 _The `<BaseUnit>` definitions can be utilized for different purposes (the following application examples are optional and a tool may also completely ignore the `Unit` definitions):_
 
-Signal connection check::
+Variable connection check::
 +
-_When two signals v1 and v2 are connected together, and on at least one of the signals no `<BaseUnit>` element is defined, then the connection equation "v2 = v1" holds (if v1 is an <<output>> of an FMU and v2 is an <<input>> of another FMU, with `fmi3Get{VariableType}` the value of v1 is inquired and used as value for v2 by calling `fmi3Set{VariableType}`)._
+_When two variables v1 and v2 are connected together, and on at least one of the variables no `<BaseUnit>` element is defined, then the connection equation "v2 = v1" holds (if v1 is an <<output>> of an FMU and v2 is an <<input>> of another FMU, with `fmi3Get{VariableType}` the value of v1 is inquired and used as value for v2 by calling `fmi3Set{VariableType}`)._
```

Comment:
Suggestion: "When an output of an FMU is connected with an input of another FMU,  and on at least one of these variables no `<BaseUnit>` element is defined, ..."

Reply:
I rewrote this entire sentence, please check! Thank you!